community/kernel/src/main/java/org/neo4j/kernel/impl/api/KernelTransactions.java

```diff
@@ -301,10 +300,6 @@ KernelTransactionHandle createHandle( KernelTransactionImplementation tx )
 
     private void assertRunning()
     {
-        if ( availabilityGuard.isShutdown() )
-        {
-            throw new DatabaseShutdownException();
-        }
```

Comment:
I think `KernelTransactionsTest.exceptionWhenStartingNewTransactionOnShutdownInstance` is failing because of this change.